e2e/visual/no-theme.spec.js

```diff
@@ -0,0 +1,84 @@
+const { expect } = require('@playwright/test');
+
+const { test: axeTest } = require('../axe-test');
```

Comment:
[I think we could just make a generic fixture like I did here on Tasklist](https://github.com/camunda/tasklist/blob/master/client/e2e/test-fixtures.ts)

Reply:
Thanks for the hint, I adjusted it via https://github.com/bpmn-io/form-js/pull/693/commits/0f3152ae77c91207f7ee2c658bb45b72a409effc 👍